koji/collection-mash-split.py

```diff
@@ -160,6 +168,32 @@ def createrepo(self, path, comps=None, checksum=None):
         self.logger.debug("running %s" % cmd)
         kobo.shortcuts.run(cmd)
 
+    def inject_modulemd_yml(self, path, modulemd_yaml, modulemd_version=0):
+        """Generate modular metadata and inject it.
+
+        @param path: path to repository
+        @type path: str
+        @param modulemd_yaml: metadata template
+        @type modulemd_yaml: str
+        """
+        # The epoch number in the NEVRA string is mandatory per spec
+        # https://github.com/fedora-modularity/libmodulemd/blob/main/yaml_specs/modulemd_stream_v2.yaml#L668
+        # So we can't just use `%{nevra}` here
+        cmd = "rpm --query --package {}/*rpm".format(path)
+        cmd += " --queryformat='%{name}-%{epochnum}:%{version}-%{release}.%{arch}\n'"
+        self.logger.debug("running %s" % cmd)
```

Comment:
If this is a regular Python logger, you can let it do the interpolation for you
```suggestion
        self.logger.debug("running %s", cmd)
```

Reply:
I don't know, I was just copying code from the other functions ;)

but it does look like the normal python logger, yeah,